EZEE-3335: As an Editor I would like to preview page using different … (#88)
ezsystems/user-documentation: docs/site_organization/working_with_page.md

    The rules are checked in order, so if a user belongs to more than one Segment, the first rule applies.

>     You can preview what the page looks like for which Segment:

Reviewer: ```suggestion
    You can preview the page for each of the available Segments:
```
